docs/openfaas-pro/iam/overview.md

```diff
@@ -103,7 +110,7 @@ Wildcards can be used to get multiple actions:
 
 Scope of access:
 
-Permissions can be scoped cluster wide, or to a specific namespace:
+Permissions can be scoped cluster wide, or to a specific namespace or function:
```

Comment:
Not strictly true. How can we word this better? The function scope only applies to functions, not to i.e. deleting them

Reply:
This part has been reworded to explain this better.